Start testing: Jun 09 12:40 CEST
----------------------------------------------------------
7/7 Testing: TestAVLTreeMinMax
7/7 Test: TestAVLTreeMinMax
Command: "/usr/bin/python3" "run_tests.py" "/home/ale/Documents/GitHub/IngenieriaInformaticaUCO/ED/repaso/P3avl-tree/build//test_avltree" "06_tests_compute_min_max_branch_length"
Directory: /home/ale/Documents/GitHub/IngenieriaInformaticaUCO/ED/repaso/P3avl-tree
"TestAVLTreeMinMax" start time: Jun 09 12:40 CEST
Output:
----------------------------------------------------------
Test running: 06_tests_compute_min_max_branch_length/00 ... [32mTest passed.[0m
Test running: 06_tests_compute_min_max_branch_length/01 ... [31mTest finished with error code -6[0m
test_avltree: /home/ale/Documents/GitHub/IngenieriaInformaticaUCO/ED/repaso/P3avl-tree/avltree_imp.hpp:518: void AVLTree<T>::insert(const T&) [with T = int]: Assertion `current_exists()' failed.

Test running: 06_tests_compute_min_max_branch_length/02 ... [31mTest finished with error code -6[0m
test_avltree: /home/ale/Documents/GitHub/IngenieriaInformaticaUCO/ED/repaso/P3avl-tree/avltree_imp.hpp:518: void AVLTree<T>::insert(const T&) [with T = int]: Assertion `current_exists()' failed.

Test running: 06_tests_compute_min_max_branch_length/03 ... [31mTest finished with error code -6[0m
test_avltree: /home/ale/Documents/GitHub/IngenieriaInformaticaUCO/ED/repaso/P3avl-tree/avltree_imp.hpp:518: void AVLTree<T>::insert(const T&) [with T = int]: Assertion `current_exists()' failed.

Test running: 06_tests_compute_min_max_branch_length/04 ... [31mTest finished with error code -6[0m
test_avltree: /home/ale/Documents/GitHub/IngenieriaInformaticaUCO/ED/repaso/P3avl-tree/avltree_imp.hpp:518: void AVLTree<T>::insert(const T&) [with T = int]: Assertion `current_exists()' failed.

Test running: 06_tests_compute_min_max_branch_length/05 ... [31mTest finished with error code -6[0m
test_avltree: /home/ale/Documents/GitHub/IngenieriaInformaticaUCO/ED/repaso/P3avl-tree/avltree_imp.hpp:518: void AVLTree<T>::insert(const T&) [with T = int]: Assertion `current_exists()' failed.

1/6 tests passed.
<end of output>
Test time =   0.99 sec
----------------------------------------------------------
Test Failed.
"TestAVLTreeMinMax" end time: Jun 09 12:40 CEST
"TestAVLTreeMinMax" time elapsed: 00:00:00
----------------------------------------------------------

End testing: Jun 09 12:40 CEST
